src/applications/gi/reducers/calculator.js

```diff
@@ -65,35 +65,48 @@ export default function (state = INITIAL_STATE, action) {
       if (__BUILDTYPE__ !== 'production') {
 
         if (field === 'yellowRibbonDegreeLevel') {
-          const {
-            yellowRibbonPrograms
-          } = state;
-
-          // make an array of unique values
-          const yellowRibbonDivisionOptions = yellowRibbonPrograms.length > 0 ?
-            [...new Set(yellowRibbonPrograms
-              .filter(program => program.degreeLevel === value)
-              .map(program => program.divisionProfessionalSchool))] :
-            [];
-
-          const {
-            contributionAmount: yellowRibbonAmount,
-            numberOfStudents: yellowRibbonMaxNumberOfStudents,
-            index: yellowRibbonProgramIndex
-          } = yellowRibbonPrograms
-            .find(program =>
-              program.degreeLevel === value &&
-              program.divisionProfessionalSchool === yellowRibbonDivisionOptions[0]);
-
-          newState = {
-            ...newState,
-            yellowRibbonAmount,
-            yellowRibbonDivisionOptions,
-            yellowRibbonDivision: yellowRibbonDivisionOptions[0],
-            yellowRibbonProgramIndex,
-            yellowRibbonMaxAmount: yellowRibbonAmount,
-            yellowRibbonMaxNumberOfStudents
-          };
+          if (value === 'Enter an amount') {
```

Comment:
Stakeholder wanted something to make it obvious that the user doesn't have to select an amount from the drop down. Open to feedback on this approach (we don't have a UX to assist with this adjustment).